config-linux.md

```diff
@@ -147,7 +147,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/random`][random.4]
 * [`/dev/urandom`][random.4]
 * [`/dev/tty`][tty.4]
-* [`/dev/console`][console.4]
+* [`/dev/console`][console.4] is setup if terminal is enabled in the config by bind mounting the pseudoterminal slave to /dev/console.
```

Comment:
>     We don't want to require bind-mounting the slave over this. Let the runtime point /dev/console to a place it considers appropriate for logging system messages, which may not be the pseudoterminal slave. In fact, I'd expect the validation for /dev/console to be “Is the major 5, the minor 1, and the type ‘char’?”, which is not going to be satisfied by bind-mounting the slave.

That validation does not work inside containers. That particular major and minor number is reserved for consoles that are created as part of initial bootup (think **physical** console) on Linux. I think that it is entirely sufficient to just state that `/dev/console` should be a valid place to log system messages (which is what you suggested).